omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -166,6 +166,18 @@ public void delete(String uuid, String reason, RequestContext context) throws Re
 		delete(delegate, reason, context);
 	}
 	
+	/**
+	 * @see org.openmrs.module.webservices.rest.web.resource.api.Deletable#unDelete(java.lang.String,
+	 *      org.openmrs.module.webservices.rest.web.RequestContext)
+	 */
+	@Override
+	public Object unDelete(String uuid, RequestContext context) throws ResponseException {
```

Comment:
Didn't i see comments about renaming these methods?

Reply:
@dkayiwa , sorry for the late response, yes you did see the comments and I made the changes in this second commit cd12521 after the first commit (054593f). I think I have to squash the commits into one. But just for reference purposes  I left this out.

Reply:
@Ruhanga can you please squash your commits into one?

Reply:
Thank you @samuelmale , commits have been squashed into one.